autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -277,7 +272,6 @@ mod tests {
         let msg = ClientMessage::Hello {
             uaid: Some("".to_owned()),
             channel_ids: None,
-            use_webpush: Some(true),
```

Comment:
It's maybe worth keeping an additional test that still sends `use_webpush` via `json!` even though the integration test continues to sending it.